src/videojs-hls.js

```diff
@@ -892,6 +892,7 @@ var filterBufferedRanges = function(predicate) {
     // report a fully empty buffer until SourceBuffers have been created
     // which is after a segment has been loaded and transmuxed.
     if (!this.mediaSource ||
+        !this.mediaSource.mediaSource_ ||
```

Comment:
I don't think we should be looking at `this.mediaSource.mediaSource_` in this project at all (which I know the next line is violating). Can this be fixed in contrib-mse?